tests/functional/test_voting_escrow.py

```diff
@@ -1,13 +1,50 @@
 from pytest import approx
 import ape
+from ape import chain
 
 H = 3600
 DAY = 86400
 WEEK = 7 * DAY
-MAXTIME = 126144000
+MAXTIME = 126144000 // WEEK * WEEK
 TOL = 120 / WEEK
 
 
+def test_over_four_years(chain, accounts, yfi, ve_yfi):
```

Comment:
can we also add a test for the minimum lock time to check balances and assumptions are correct for a user based on these latest changes?

Reply:
Can you add it? I am trying to get more people involved in coding this here, this is a simple test, please do it.

Reply:
The test exists already check line 155 bob creates a lock for only a week

Reply:
> Can you add it? I am trying to get more people involved in coding this here, this is a simple test, please do it.

I will finish helping with reviews first to prioritize getting context on code and audit changes first before i dive into adding tests, i will start helping on testing side to get more coverage